governance-exercises/Executive Vote 02.md

```diff
@@ -0,0 +1,162 @@
+
+#### DAI Market Parameter Changes
+
+- DAI market `Loan To Value` will be set to 0.01%.
+- DAI market `Liquidation Threshold` will be set to 0.01%.
+
+Since the sDAI market is frozen, the only way to short is through the DAI market. These proposed changes will effectively remove DAI as a collateral type, ensuring there won’t be a situation where the market is at 100% utilization and liquidations on the short positions need to be done.
+
+#### WETH Market Parameter Changes
+
+- WETH market `reserveFactor` will be set to 5%. This will reduce the protocol fee taken on the ETH market to give suppliers a larger APY.
+- WETH market `variableRateSlope1` will be set to 3%. This will ensure a healthier utilization near the kink to improve supplier return and incentivize more deposits.
+
+### Launch Project Funding Distribution
+
+*Authorization*: [Accessibility Scope Artifact](https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget)\
+*Reasoning & authorizing party*: Accessibility Scope Facilitator's [Forum post](http://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468)
+
+[Launch Project](http://forum.makerdao.com/t/the-launch-project-and-scope-artifact-update-proposal/21048?) funding was previously approved as part of the ratification of the Accessibility Scope. The Accessibility Scope Facilitator now seeks to establish a wallet to hold 2 million DAI of Launch Project funds. This will allow the Facilitator to more efficiently access funds to carry out work related to the MakerDAO rebranding, Easy Governance Frontend and other growth/marketing expenses. If this executive proposal passes, the following action will occur:
+
+- **2 million DAI** of Launch Project funds will be transferred to wallet [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F), controlled by the Accessibility Scope Facilitator.
+
+### New Silver (RWA002-A) Vault - Update RWA Agreement Parameter
+
+*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmaU1eaD#vote-breakdown)\
+*Reasoning*: Strategic Finance Core Unit's [Forum post](http://forum.makerdao.com/t/rwa-002-new-silver-restructuring-risk-and-legal-assessment/21417)\
+*Additional authorizing party*: [Dewiz](http://forum.makerdao.com/t/rwa-002-new-silver-restructuring-risk-and-legal-assessment/21417/15)
+
+The restructuring and parameter changes of [New Silver’s existing vault](https://forum.makerdao.com/t/ns-drop-collateral-onboarding-risk-evaluation/5418) have necessitated updates to the underlying legal documentation. In turn, this requires updating the [RWA Agreement](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) parameter that stores an IPFS hash of the legal documentation. If this executive proposal passes, the following action will occur:
```

Comment:
I think the other parameter changes haven't taken place yet, and that updating the doc is required before updating the parameters (for some reason that escapes me, but hopefully makes sense to someone in the chain of responsibility here...)